README.md

```diff
@@ -218,10 +218,6 @@ async-std = { version = "1", features = [ "attributes" ] }
 # Tokio:
 sqlx = { version = "0.5", features = [ "runtime-tokio-native-tls" , "postgres" ] }
 tokio = { version = "1", features = ["full"] }
-
-# Actix-web:
-sqlx = { version = "0.5", features = [ "runtime-actix-native-tls" , "postgres" ] }
-actix-web = "3"
```

Comment:
To avoid confusion we should probably keep this since `#[actix_web::main]` appears to still be preferred: https://docs.rs/actix-web/4.1.0/actix_web/rt/index.html#running-actix-web-using-tokiomain

Reply:
These version numbers have been updated on `main` already, by the way.